cpp/include/raft/util/cuda_utils.cuh

```diff
@@ -530,6 +540,11 @@ template <typename Type>
 struct Sum {
   HDI Type operator()(Type a, Type b) { return a + b; }
 };
+
+template <typename Type>
+struct Max {
+  HDI Type operator()(Type a, Type b) { return myMax(a, b); }
```

Comment:
For some reason, I had an impression that we wanted to deprecate `myXxx`-style functions in raft. Is that the case, @cjnolet ? Here, we could use `std::max`, for it being `constexpr`.

Reply:
@Nyrio do you want to create an issue for these and do them as a follow-on? I think since this PR has already been approved and run through CI (and since burndown starts tomorrow), we can go ahead and merge this as-is. What do you guys think?